### ISSUE TITLE
[DRIVER/FORMAT] COG support

### ISSUE THREAD

**Issue by wonder-sk** [lutraconsulting/input-sdk#46]
Right now cloud optimized GeoTiffs (COG) are not showing up in Input (when stored on a remote server) - presumably because we build GDAL without `curl` library.

Remote COGs use GDAL's `vsicurl` virtual file system, e.g. `/vsicurl/http://www.example.com/raster.tif`

**Comment by PeterPetrik** [comment]
@wonder-sk do you have some test case?

**Comment by wonder-sk** [comment]
```
http://even.rouault.free.fr/gtiff_test/S2A_MSIL1C_20170102T111442_N0204_R137_T30TXT_20170102T111441_TCI_cloudoptimized_512.tif
```
In QGIS it can be loaded in Data Source Manager > Raster > Protocol

**Comment by PeterPetrik** [comment]
local COG should work

**Comment by saberraz** [comment]
I propose to close this ticket and open a new one for all remote file formats using `vsicurl` (e.g. csv, geojson, COG, etc).

**Comment by wonder-sk** [comment]
Added in Input 1.3.0

@saberraz all file formats with `vsicurl` should work now, so we probably don't need any new tickets for other remote formats

**Comment by PeterPetrik** [comment]
this is PR, not merged in 1.3.0...

**Comment by wonder-sk** [comment]
ah sorry I got confused :facepalm:
